app/src/main/java/com/tpp/theperiodpurse/ui/setting/NotificationsScreen.kt

```diff
@@ -139,13 +149,24 @@ fun setAlarm(context: Context, pickedTime: LocalTime){
     }
 
     val alarmManager = context.getSystemService(Context.ALARM_SERVICE) as AlarmManager
-    val intent = Intent(context, Alarm::class.java)
+//    val intent = Intent(context, Alarm::class.java)
+
+    val freq = appViewModel.getReminderFreq()
+    lateinit var intent: Intent
+    if (freq == "Every day"){
+        intent = Intent(context, Alarm::class.java)
```

Comment:
Is 120000 for a week or a day? Looks like they are used in both `Alarm` and `WeeklyAlarm`